3_Dictionaries_part2.ipynb

```diff
@@ -58,9 +58,20 @@
   },
```

Comment:
<p>write your code in functions, this is a very very bad practice - create relevant functions and implement them so that each has its own SINGLE responsibility</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week04-ofirshan/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
<p>fixed</p>